core/payments/payments.go

```diff
@@ -189,6 +200,24 @@ func (s *service) GetSubscriptionStatus(ctx context.Context, req *pb.RpcPayments
 		}
 	}
 
+	// 5 - save RequestedAnyName to details of local identity object
+	spc, err := s.spaceService.Get(context.Background(), s.account.PersonalSpaceID())
+	if err != nil {
+		log.Error("failed to get personal space id:" + err.Error())
+		return &out, nil
+	}
+	if err = spc.Do(s.account.MyParticipantId(s.account.PersonalSpaceID()), func(sb smartblock.SmartBlock) error {
+		if ds, ok := sb.(basic.DetailsSettable); ok {
+			return ds.SetDetails(nil, []*pb.RpcObjectSetDetailsDetail{{
+				Key:   bundle.RelationKeyGlobalName.String(),
+				Value: pbtypes.String(status.RequestedAnyName),
```

Comment:
maybe some check for the status of the subscription is required here, but leave it as-is
but not sure
and I will fix it by myself later, if needed